docs/process/testing.md

```diff
@@ -0,0 +1,232 @@
+  - [ ] `Sign in to Github.com` link
+    - [ ] `Sign in` successful if valid username/email and password
+      - [ ] If 2FA activated, user sent auth code to enter 
+	- [ ] Sign in successful with active 2FA code, user goes to Configure Git
+	  - [ ] Error message if code is wrong or inactive 
+      - [ ] Error message if incorrect username/email or password
+    - [ ] Forgot link (/password_reset)
+    - [ ] `Cancel` returns to initial Welcome Flow
+    - [ ] `Sign in using your browser` opens default browser
+      - [ ] Browser login, "authorize" GitHub Desktop, “accept” link
+  - [ ] `Sign in to Enterprise` link (works with v2.8 and up)
+    - [ ] `Continue` successful if server address is valid, else error message
+      - [ ] `Sign in using your browser` opens default browser
+  - [ ] Browser login, [insert custom security measure], Authorize GitHub Desktop, “accept” link
+    - [ ] User goes to Configure Git if successful
+    - [ ] `Cancel` returns to initial Welcome Flow
+    - [ ] User served generic message if not authorized to access Enterprise server
+  - [ ] Skip step
+    - [ ] Configure Git
+      - [ ] Name and email pulled from global `.gitconfig` file, if configured
+  	- [ ] If recognized, avatar is present
+      - [ ] `Continue` okay if fields populated or blank
+  	- [ ] Valid login credentials from dotcom or Enterprise carried through 
+  - [ ] Usage Data
+    - [ ] Checked by default; user can uncheck
+      - [ ] Clicking `Finish`, and user is signed in successfully to Desktop
+    - [ ] `Cancel` returns to initial Configure Git page
+
+### Repositories landing page; default no repositories 
+  - [ ] Create New Repository (Mac: ⌘N; Windows: Ctrl+N)
+    - [ ] Modal opens with name, path (choose option), readme (unchecked), git ignore, license. Name and path mandatory.
+    - [ ] `Create Repository` button adds new repo, which is added to Repo list
+    - [ ] `Cancel` button does not save any changes made; modal closed
+  - [ ] Clone a Repository (Mac: ⇧⌘O; Windows: Ctrl+Shift+O)
+    - [ ] Enter valid URL or username/repo, else error message
+    - [ ] Valid path can be entered or selected, else error message
+    - [ ] `Clone` creates repo at selected path
+      - [ ] Repo added to Repo list
+    - [ ] `Cancel` closes modal, no repo cloned
+  - [ ] Add a Local Repository (Mac: ⌘O; Windows: Ctrl+O)
+    - [ ] Valid path can be entered or selected
+    - [ ] `Add repository` activated if repo path exists
+      - [ ] Repo added to Repo list
+    - [ ] `Cancel` closes modal, no repo added
+  - [ ] Drag and drop repository
+    - [ ] User can drag existing local repository into Desktop
+      - [ ] Successful attempt adds repo to Repo list; else error message
+
+### Application
+  - [ ] Minimize, Maximize, Close buttons work in top nav
+  - [ ] Double-clicking local desktop icon opens the application
+  - [ ] Double clicking top nav bar toggles full-screen / last used screen-size
+
+### GitHub Desktop menu
```

Comment:
Maybe call out this is macOS only? Or maybe reorganize this based around the features themselves?

On Windows:
 - the `About` top-level menu is home to the `About GitHub Desktop` functionality
 - the `File` top-level menu is home to the `Options` functionality (different name) 
 - no options for installing command line tool (we do this automagically)
 - `File` | `Exit`